ql/src/java/org/apache/hadoop/hive/ql/stats/ColStatsProcessor.java

```diff
@@ -231,6 +234,51 @@ public int persistColumnStats(Hive db, Table tbl) throws HiveException, MetaExce
   @Override
   public void setDpPartSpecs(Collection<Partition> dpPartSpecs) {
   }
+  
+  private boolean isExistsNewerCommittedWriteId(){
+    return false;
+  }
+
+  public static boolean canSkipStatsGeneration(String dbName, String tblName, String partName,
```

Comment:
seems this is  one of the utility methods, can we consider moving it to StatsUtils.java? we need to add test as well